examples/python/debug/get_stats.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 The gRPC Authors
```

Comment:
It might be a good idea to include a section in the README about how to use `grpc_cli` to do the same thing. Most people aren't going to want to have to write Python just to debug their service.

Reply:
Recommendation to `grpc_cli` added to the last section.

We didn't distribute `grpc_cli`'s binary, so people will have to figure out how to build it... But anyway, it's a good tool.